db/deploy/tables/gis_data.sql

```diff
@@ -0,0 +1,50 @@
+-- Deploy ccbc:tables/gis_data to pg
+
+begin;
+
+create table ccbc_public.gis_data(
+  id integer primary key generated always as identity, 
+  json_data jsonb not null default '{}'::jsonb
```

Comment:
I imagined the table would have a row per array element of the gis_data, although I like this better since we can get the new gis_data easily